tests/impl.cpp

```diff
@@ -3502,7 +3502,29 @@ result_t test_mm_srli_epi32(const SSE2NEONTestImpl &impl, uint32_t i)
 
 result_t test_mm_slli_epi64(const SSE2NEONTestImpl &impl, uint32_t i)
 {
-    return TEST_UNIMPL;
+    const int64_t *_a = (const int64_t *) impl.mTestIntPointer1;
+    const int count1 = 3; /* imm8 <= 63 */
```

Comment:
I think it can be simplified as
```c
#if define(__clang__)
int count = i % 64;
#else 
int count = i % 70; // or other value greater than 64
#endif 
```
and the extra code below can be removed.